Sources/JobsPostgres/Migrations/CreateJobs.swift

```diff
@@ -24,11 +24,18 @@ struct CreateJobs: PostgresMigration {
                 id uuid PRIMARY KEY,
                 job bytea,
                 status smallint,
-                lastModified TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+                last_modified TIMESTAMPTZ DEFAULT NOW()
```

Comment:
Can we remove the variable renaming. If we are going to rename the variables we should do it in another PR